geometry_msgs/Pose[] targets
std_msgs/String group_name
std_msgs/String plan_execute
